.env.defaults

```diff
@@ -41,6 +42,16 @@ CYPRESS_PAGE_SIZE_CONTRACTORS=10
 PAGE_SIZE_AGENTS=10
 CYPRESS_PAGE_SIZE_AGENTS=10
 
+### Cypress auth tokens
```

Comment:
Maybe a comment to the effect that "these will require populating in .env.local" or something along those help-a-future-developer-out kind of problems

Reply:
I have added some context